components/preview/package.json

```diff
@@ -20,15 +15,19 @@
   "scripts": {
     "clean": "rimraf build dist lib* types temp tmp *.tsbuildinfo",
     "compile-es6": "tsc -p ./tsconfig.json",
+    "compile-es6-watch": "npm run compile-es6 -- -w",
     "bundle": "rollup -c ../rollup.config.js",
+    "bundle-watch": "npm run bundle -- -w",
     "build": "npm run compile-es6 && npm run bundle",
     "update": "npx npm-check-updates -u -t minor"
   },
   "dependencies": {
     "@hpcc-js/wc-core": "^0.2.0",
     "@hpcc-js/wc-editor": "^0.2.0",
+    "@hpcc-js/wc-layout": "^0.1.0",
```

Comment:
This doesn't match the `"@hpcc-js/wc-layout"` version in `layout/package.json`

Reply:
Will fix.

Reply:
Resolved